docs/content/_index.md

```diff
@@ -638,6 +638,12 @@ On macOS (64-bit):
 curl -L -o opa https://github.com/open-policy-agent/opa/releases/download/{{< current_version >}}/opa_darwin_amd64
 ```
 
+or using [Homebrew](https://brew.sh/)
+
+```shell
+brew install opa
```

Comment:
One potential issue here is that the doc content is versioned on the website, but this command would always install the latest opa release available. Can we update this to something like `brew install opa@{{ <current_version> }}` and have it work as expected? (Note: `current_version` also uses aliases `latest` and `edge`)

I'm also not sure how quickly it gets updated after we cut releases (AFAIK its not something that is currently part of the release process).

My main concern is that while this might be a pretty valid way to install it we want to make sure that this guide minimizes any issues a user might have.

Reply:
We've talked about this in the past and I ended up filing an issue to create an Installation page. See #1559. I would put this under the Operations section.